Tests/BSONTests/DocumentTests.swift

```diff
@@ -252,60 +233,43 @@ final class DocumentTests: MongoSwiftTestCase {
     }
 
     func testEquatable() {
-        expect(["hi": true, "hello": "hi", "cat": 2] as Document)
-            .to(equal(["hi": true, "hello": "hi", "cat": 2] as Document))
+        expect(["hi": true, "hello": "hi", "cat": 2] as BSONDocument)
+            .to(equal(["hi": true, "hello": "hi", "cat": 2] as BSONDocument))
     }
 
-    func testRawBSON() throws {
-        let doc = try Document(fromJSON: "{\"a\" : [{\"$numberInt\": \"10\"}]}")
-        let fromRawBSON = try Document(fromBSON: doc.rawBSON)
-        expect(doc).to(equal(fromRawBSON))
-    }
+    // TODO: SWIFT-930: unskip
+    // func testRawBSON() throws {
+    //     let doc = try BSONDocument(fromJSON: "{\"a\" : [{\"$numberInt\": \"10\"}]}")
+    //     let fromRawBSON = try BSONDocument(fromBSON: doc.rawBSON)
+    //     expect(doc).to(equal(fromRawBSON))
+    // }
 
     func testCopyOnWriteBehavior() {
```

Comment:
this test has been largely simplified, as we can now assume that ByteBuffer correctly handles sharing storage for us. in the past we implemented copy-on-write ourselves so this test was checking that storage would be correctly copied on mutations.